.gitlab/workflows/renovate.yml

```diff
@@ -0,0 +1,21 @@
+renovate:
+  cache:
+    key: ${CI_COMMIT_REF_SLUG}-renovate
+    paths:
+      - renovate/cache/renovate/repository/
+  image: renovate/renovate:37.202.2
+  rules:
```

Comment:
docs is needed on how to set up scheduled tasks for renovate.

Reply:
Good point, I have added that to the task list in #188.